### ISSUE TITLE
give priority to non-peoplesopen traffic on private hotspot

### ISSUE THREAD

**Issue by jhpoelen** [sudomesh/bugs#2]
current - home routers can be configured to cap peoplesopen traffic.

suggested by Gerald - home router should be configured to give private network traffic priority over peoplesopen network traffic. This way, owners can be guaranteed to have full access to their internet bandwidth even through peoplesopen traffic is present.

**Comment by sierkje** [comment]
Would this need to be fixed in [sudomesh/sudowrt-firmware](https://github.com/sudomesh/sudowrt-firmware)?

**Comment by Juul** [comment]
@sierkje yes, we would probably have to create a [qos](https://wiki.openwrt.org/doc/uci/qos) configuration file in [/etc/config/](https://github.com/sudomesh/sudowrt-firmware/tree/master/files/etc/config) or if the UCI interface does not provide the needed control then it could be added in the form of `tc` commands in `/etc/init.d/meshrouting`

**Comment by paidforby** [comment]
As far as I can imagine, we have no way of guaranteeing a QoS for a node operator's private network. If there is an internet connection over your home node's private network, then you probably have a home wireless network provided by an ISP home router. This network is completely separate and cannot be managed by a peoplesopen node.

For example a node operate may see three SSIDs in their home with associated IPs,
ATT2WIREfinity: 10.0.0.1 (managed by ISP home router)
pplsopenadmin: 172.30.0.1 (managed by peoplesopen node)
peoplesopen.net 65.130.65: 100.65.130.65 (managed by peoplesopen node)

A better way of handing traffic priortization would be to implement a rate limit or QoS the port on the upstream ISP home router that is providing internet connectivity to the home node. This means there is no general solution to this issue and must be handled case by case by node installers.
  
In my opinion, the existence of the private network is misleading and confusing to most node operators, unless we start relaying private network traffic (which is possible now that we are preferring airOS). Otherwise, I like null's idea that the private network should just be a blackhole to the admin dashboard. 

See related discussion about private networks here, https://github.com/sudomesh/sudowrt-firmware/issues/131#issuecomment-382556437

**Comment by paidforby** [comment]
@bennlich suggested reframing this issue not in terms of public vs. private networks, but as node operators devices vs. random devices. In this sense, a node operator could register their devices' MAC addresses in whitelist that would prioritize them over random devices. It's unclear if this is possible with [qos-scripts](https://wiki.openwrt.org/doc/uci/qos) or with [tc](https://linux.die.net/man/8/tc). If possible, this solution could encourage node operators to use their peoplesopen SSID over their private network
and would also work well with the idea of blackholing the private network to the admin dashboard.
